modules/scanning-delegate-role/main.tf

```diff
@@ -467,6 +467,36 @@ data "aws_iam_policy_document" "scanning_worker_dspm_policy_document" {
       values   = ["s3.*.amazonaws.com"]
     }
   }
+
+  statement {
+    sid    = "DatadogAgentlessScannerRDSStartExportTask"
+    effect = "Allow"
+    actions = [
+      "rds:StartExportTask"
+    ]
+    resources = [
+      "arn:${data.aws_partition.current.partition}:rds:*:*:cluster:*",
+      "arn:${data.aws_partition.current.partition}:rds:*:*:cluster-snapshot:*",
+      "arn:${data.aws_partition.current.partition}:rds:*:*:snapshot:*",
+    ]
+  }
```

Comment:
We could add this condition to allow excluding snapshot from being exported:

```
    condition {
      test     = "StringNotEquals"
      variable = "aws:ResourceTag/DatadogAgentlessScanner"
      values   = ["false"]
    }
```

Reply:
Thank you! Added